README.md

```diff
@@ -35,4 +35,18 @@ It is extended for the following uses:
 
 ![](docs/cnap-uses.png)
 
+## 3. Building
+
+The [build script](tools/docker_image_manager.sh) helps to build docker images of our microservices. For instance, to build all docker images, use the following command:
```

Comment:
The provided [build script](./tools/docker_image_manager.sh) simplifies the process of building Docker images for our microservices.

Reply:
Thanks, fixed.